src/main.cpp

```diff
@@ -4293,9 +4327,14 @@ static bool AcceptBlock(const CBlock& block, CValidationState& state, const CCha
         if (fTooFarAhead) return true;      // Block height is too high
     }
 
-    // See method docstring for why this is always disabled
+    // See method docstring for why this is always disabled. The only caller of
+    // AcceptBlock (ProcessNewBlock) will fully check proofs when it eventually
+    // delegates to `ConnectTip` and thereafter `ConnectBlock`. This
+    // arrangement is a bit fragile and should be reconsidered.
```

Comment:
This now both refers to the method docstring and then partially duplicates it. Please de-duplicate (ideally into the method docstring).